### PR TITLE
Selenium turbal

### PR DESCRIPTION
**Pull request by TurbalY**

### CODE REVIEW

File: Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```
Comment (romanukhaav) [inline Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/Program.cs:12]: We no need an empty Main method. We should just change the Output type of the project to ClassLibrary

image

File: Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/SeleniumFirst.cs [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+           // driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+           // driver.FindElement(By.CssSelector("#search > input")).Clear();
+           // driver.FindElement(By.CssSelector("#search > input")).SendKeys("Nokia" + Keys.Enter);
+            // driver.FindElement(By.CssSelector("#search > My Account"));
+            //driver.FindElement(By.CssSelector("#search > login")).SendKeys(Keys.Enter);
+            // driver.FindElement(By.CssSelector("#search > E-Mail Address")).SendKeys("marianta1");
+            // driver.FindElement(By.CssSelector("#search > Password")).SendKeys("marianta1" + Keys.Enter);
+           // driver.FindElement(By.CssSelector("#search > input")).Clear();
+           // driver.FindElement(By.CssSelector("#search > input")).SendKeys("iPhone" + Keys.Enter);
+            driver.Quit();
```
Comment (romanukhaav) [inline Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/SeleniumFirst.cs:37]: We should have some assertions in the test.

File: Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/SeleniumFirst.cs [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+                    m_driver.Manage().Window.Maximize();
+
+                    IWebElement course = m_driver.FindElement(By.XPath("//*[@id='awf_field-91977689']"));
+
+                    var selectTest = new SelectElement(course);
+                    // Select a value from the dropdown				
+                    selectTest.SelectByValue("sap-abap");
+                    m_driver.Quit();
+
+
```
Comment (romanukhaav) [inline Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/SeleniumFirst.cs:56]: Could you please perform some cleanup and remove all commented code?

**Review by romanukhaav — CHANGES_REQUESTED** [review]

**Comment by TurbalY** [comment]
Thanks a lot. I have some problem dealing with selenium driver. Local host
stop connection after 60 seconds. I 'm solving this problem .


пн, 23 трав. 2022, 10:16 користувач Anton ***@***.***> пише:

**Comment by romanukhaav** [comment]
> 
did you use the last selenium examples? I've already fixed this issue "Local host
stop connection after 60 seconds"

**Review by romanukhaav — APPROVED** [review]
It is better, but I still can't find Assertions in these tests

**Review by romanukhaav — APPROVED** [review]

**Pull request merged**